src/picongpu/CMakeLists.txt

```diff
@@ -167,33 +59,15 @@ endif(VAMPIR_ENABLE)
 option(PIC_RELEASE "Build release version, disables all runtime asserts" OFF)
 if(PIC_RELEASE)
     message(STATUS "Release version")
-
     set(CMAKE_BUILD_TYPE Release)
     add_definitions(-DNDEBUG)
-    set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS}" "-Xcompiler=-pthread")
 else(PIC_RELEASE)
     message(STATUS "Debug version")
-
     set(CMAKE_BUILD_TYPE Debug)
-    set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS}" -g "-Xcompiler=-g,-pthread")
 endif(PIC_RELEASE)
 
 
 ################################################################################
-# Find MPI
```

Comment:
We **are** using MPI directly in PIConGPU, so PIConGPU should  do a `find_package` on its own (again).